src/Interface/Modules/Render/GLWidget.h

```diff
@@ -93,10 +94,10 @@ public Q_SLOTS:
 private:
   Render::MouseButton getSpireButton(QMouseEvent* event);
 
-  Render::RendererPtr                   mGraphics          {};  ///< Interface to spire.
-  QTimer*                               mTimer             {};
-  double                                mFrameTime         {0.0};
-  bool                                  mFrameRequested    {false};
+  Render::RendererPtr                   graphics_          {};  ///< Interface to spire.
+  QTimer*                               timer_             {};
+  bool                                  frameRequested_    {false};
+  bool                                  canClickMouse_     {true};
```

Comment:
I changed these from mVarName to varName_ because it is not a part of spire. I deleted mFrameTime because it is not being used anywhere